packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -228,17 +232,18 @@ const sharedStyles = StyleSheet.create({
     },
     withIcon: {
         // The left padding for the button with icon should have 4px less padding
-        paddingLeft: 12,
+        paddingLeft: theme.padding.medium,
     },
     disabled: {
         cursor: "auto",
     },
     small: {
-        height: 32,
+        borderRadius: theme.border.radius.small,
```

Comment:
**note:** Added a borderRadius value here b/c the `khanmigo` theme overrides this value for `small` buttons.